unittest/python/test_user_type.py

```diff
@@ -7,8 +7,8 @@
 def test(mat):
   mat[:] = mat.dtype.type(10.)
   mat_copy = mat.copy()
-  assert (mat == mat_copy).all()
-  assert not (mat != mat_copy).all()
+  assert((mat == mat_copy).all())
```

Comment:
`assert`(https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement) is a statement, not a function: it doesn't need parenthesis

Reply:
I removed this commit.